kaldb/src/main/java/com/slack/kaldb/logstore/search/LogIndexSearcherImpl.java

```diff
@@ -243,7 +234,9 @@ private Query buildQuery(
     if (queryStr.length() > 0) {
       queryBuilder.add(buildQueryParser().parse(queryStr), Occur.MUST);
     }
-    return queryBuilder.build();
+    BooleanQuery query = queryBuilder.build();
```

Comment:
Need to add span tag here and not parent method because we return `Query` , while all info is as a `BooleanQuery`